cloud-service-providers/google-cloud/vertexai/python/nim-vertexai.ipynb

```diff
@@ -123,7 +105,7 @@
    },
    "outputs": [],
    "source": [
-    "import google.cloud.aiplatform as aiplatform\n",
+    "import google.cloud.aiplatform as aiplatform # redundant?\n",
```

Comment:
@FortunaZhang I'm not sure if importing this package here makes sense as there's another cell after importing:

`from google.cloud import aiplatform`

it seems a bit redundant but please lmk your thoughts

Reply:
Yes this is same as the other import below.  

from google.cloud import aiplatform
aiplatform.init(project=project_id, location=region, staging_bucket=bucket_url)